File: Iniciante/URI 1007.hs
main :: IO ()
main = do
    a' <- getLine
    b' <- getLine
    c' <- getLine
    d' <- getLine
    let a = read a' :: Int
    let b = read b' :: Int
    let c = read c' :: Int
    let d = read d' :: Int
    let diferenca = a * b - c * d
    putStrLn $ "DIFERENCA = " ++ show diferenca

File: Iniciante/URI 1007.pas
var
    a : int64;
    b : int64;
    c : int64;
    d : int64;
    diferenca : int64;

begin
    read(a);
    read(b);
    read(c);
    read(d);
	
    diferenca := a * b - c * d;
	
    writeln('DIFERENCA = ', diferenca);
end.